docs2/site/docs/migrations/migration8.md

```diff
@@ -1052,6 +1052,20 @@ Sample persisted document request:
 }
 ```
 
+#### Caching (v8.3.0+)
+
+The persisted document handler does not provide caching by default. You may implement your own caching mechanism
+within the `GetQueryAsync` method to cache the query strings based on the document identifier. Alternatively, you
+may add the `.UseMemoryCache()` method to enable in-memory caching. Be sure to call `UseMemoryCache` before calling
```

Comment:
It is worth mentioning that the `GraphQL.MemoryCache` package is required.

Reply:
updated